src/modules/ordersTable/containers/OrdersTableWidget/index.tsx

```diff
@@ -46,12 +46,12 @@ const ContentWrapper = styled.div`
   width: 100%;
 `
 
-export function OrdersTableWidget() {
+export function OrdersTableWidget({ customOrders }: { customOrders?: ParsedOrder[] }) {
```

Comment:
I agree, this is not the best solution.
The task is:
 - `<OrdersTableWidget>` component displays all orders from order-book by default
 - we need to display TWAP orders in the orders table as well

How it solved now:
 - those twap orders are not "real", so we emulate them using `parsePendingTwapOrder`
 - we add those emulated twap orders using `customOrders`
 
 I have only one other option:
 - like [here](https://github.com/cowprotocol/cowswap/commit/f4338bebe7306fc6925b512819cc0e411566b75c) I can add them at the very beginning in `useGpOrders`
 - since I already created a crime here, it's better to be consistent :)

Reply:
The fix: https://github.com/cowprotocol/cowswap/pull/2597/commits/a4460d8746c81d67d1629cde5eb56c83b359ec55

Reply:
Nice!